libs/ledger-live-common/src/families/cardano/cli-transaction.ts

```diff
@@ -2,7 +2,7 @@ import { Account, AccountLike } from "@ledgerhq/types-live";
 import invariant from "invariant";
 import flatMap from "lodash/flatMap";
 import { getAccountCurrency } from "../../account";
-import { TokenAccount } from "../solana/api/chain/account/token";
```

Comment:
Cardano depends on Solana?!

Reply:
It surprised me too.
The fix will be part of Cardano modularization...